pontoon/base/migrations/0120_bug_1423679_update_plurals_20180417_1135.py

```diff
@@ -0,0 +1,259 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.11 on 2018-04-17 11:35
+from __future__ import unicode_literals
+
+import copy
+
+from django.db import migrations
+from fluent.syntax import FluentParser, FluentSerializer, ast
+
+# Needed for calculate_stats()
+from pontoon.base.models import TranslatedResource
+
+
+parser = FluentParser()
+serializer = FluentSerializer()
+
+
+def update_plurals(apps, schema_editor):
```

Comment:
Maybe it would be better to implement this part as a `ModelOperation`? Because that would give us the ability to re-use this code in new migrations.
Examples:
https://docs.djangoproject.com/en/2.0/_modules/django/db/migrations/operations/models/